Delcam.ProductInterface.PowerMILL.Test/PMAutomationTest.cs

```diff
@@ -247,5 +247,15 @@ public void GetListOfPmComObjectsTest()
             //Quit
             automation2.Quit();
         }
+
+        [Test]
+        public void UndrawAllTest()
+        {            
+            MyTestInitialize();
+            var testPattern = _powerMill.ActiveProject.Patterns.CreatePattern(Files.TestFiles.CurvesFiles);
+            testPattern.IsVisible = true;
+            _powerMill.UndrawAll();
+            //Assert.False(testPattern.IsVisible);                     
```

Comment:
I don't know either...  If you want to do "Assert.Pass()" then I would be ok with that!